### PR TITLE
chore(snippet-bot): upgrade snippet-bot to Probot11

### PR DESCRIPTION
**Pull request by tmatsuo**
Superseding #1385

### CODE REVIEW

File: packages/snippet-bot/test/snippet-bot.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('snippet-bot', () => {
   let invalidateCacheStub: sinon.SinonStub;
 
   beforeEach(() => {
-    probot = createProbot({
```
Comment (sofisl) [inline packages/snippet-bot/test/snippet-bot.ts:55]: You can leave this as createProbot, just add `overrides`: {github token...}

File: packages/snippet-bot/package.json [hunk 2]
```diff
@@ -29,9 +29,9 @@
   },
   "dependencies": {
     "@google-cloud/storage": "^5.4.0",
-    "gcf-utils": "^6.2.0",
-    "node-fetch": "^2.6.0",
+    "gcf-utils": "^7.1.1",
     "minimatch": "^3.0.4",
+    "node-fetch": "^2.6.0",
```
Comment (sofisl) [inline packages/snippet-bot/package.json:34]: Do you end up using this?

File: packages/snippet-bot/test/snippet-bot.ts (hunk 1 above)
Comment (tmatsuo) [reply packages/snippet-bot/test/snippet-bot.ts:55]: `createProbot` is deprecated anyways.

File: packages/snippet-bot/package.json (hunk 2 above)
Comment (tmatsuo) [reply packages/snippet-bot/package.json:34]: Thanks, although I think it's a good idea, we should tackle that in another PR in my opinion. WDYT?

**Review by sofisl — APPROVED** [review]

**Pull request merged**